checkstyle/import-control.xml

```diff
@@ -240,6 +240,7 @@
       <allow pkg="org.apache.kafka.test" />
       <allow pkg="org.apache.kafka.timeline" />
       <subpackage name="metrics">
+        <allow pkg="com.yammer.metrics"/>
```

Comment:
I recall we removed yammer metrics from the last pr, do we also want to do that here?

Reply:
Or do we have this just to cover the existing generic metrics?

Reply:
yeah, unfortunately we need this for the existing metrics

Reply:
Are we planning not to add any of the new sensor metrics though? I assume that's what the description is referring to when it says we need to wait for other things to get the sensors.

Reply:
it's ready

Reply:
ooof. I'm too slow at reviewing :(